project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -33,34 +34,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // If event DNE, sends 404 ERR to frontend
     if (!eventOptional.isPresent()) {
-        response.sendError(HttpServletResponse.SC_NOT_FOUND);
+      response.sendError(HttpServletResponse.SC_NOT_FOUND);
     } else {
-        Event event = eventOptional.get().toBuilder().setId(eventId).build();
-        response.setContentType("text/html;");
-        response.getWriter().println(new Gson().toJson(event));
+      Event event = eventOptional.get().toBuilder().setId(eventId).build();
+      response.setContentType("text/html;");
+      response.getWriter().println(new Gson().toJson(event));
     }  
   }
 
-  /** Posts new created event to database and redirects to page with created event details*/
+  /** Posts new created event to database and redirects to page with created event details */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String name = request.getParameter("name");
     String[] parsedDate = request.getParameter("date").split("/");
-    Date date = Date.fromYearMonthDay(Integer.parseInt(parsedDate[2]), Integer.parseInt(parsedDate[1]), Integer.parseInt(parsedDate[0]));
+    Date date =
+         Date.fromYearMonthDay(
+            Integer.parseInt(parsedDate[2]),
+            Integer.parseInt(parsedDate[0]),
+            Integer.parseInt(parsedDate[1]));
     String description = request.getParameter("description");
     String location = request.getParameter("location");
-    Set<String> labels = Collections.unmodifiableSet(new HashSet<>(
-        Arrays.asList("None"))); // hardcoded for now, we need to create label pool first
+    Set<String> labels = 
+        Collections.unmodifiableSet(
+            new HashSet<>(
+                Arrays.asList("None"))); // hardcoded for now, we need to create label pool first
 
-    /** TO DO: Replace with current logged in user after PR #43 pushed*/
+    /** TO DO: Replace with current logged in user after PR #43 pushed */
     String NAME = "Bob Smith";
     String EMAIL = "bobsmith@example.com";
     User host = new User.Builder(NAME, EMAIL).build();
     Event event = new Event.Builder(name, description, labels, location, date, host).build();
     DatabaseWrapper dbWrapper = new DatabaseWrapper("step-188-instance", "event-organizer-db");
```

Comment:
Use Database constants in DatabaseConstants.java

Reply:
With SpannerTasks being static this line can probably just be taken out